tests/PhpSpreadsheetTests/Reader/XlsxTest.php

```diff
@@ -32,4 +32,18 @@ public function testLoadWithReadFilter()
             $this->assertEquals($ref, \array_slice($data[$i], 0, 10, true));
         }
     }
+
+    /**
+     * Test load Xlsx file while ignoring blank cells.
+     */
+    public function testLoadIgnoringBlankCells()
+    {
+        $filename = './data/Reader/XLSX/99_empty_cells.xlsx';
```

Comment:
Instead of adding a binary file to the repo, create it from scratch, on the fly, when testing. You probably want to use `\PhpOffice\PhpSpreadsheetTests\Functional\AbstractFunctional` for that. And this should also cover XLS to prove that the behavior is similar for both formats. See other functional tests for inspiration.